dandi/tests/fixtures.py

```diff
@@ -448,7 +444,9 @@ def bids_dandiset(new_dandiset: SampleDandiset, bids_examples: str) -> SampleDan
 
 
 @pytest.fixture()
-def bids_dandiset_invalid(new_dandiset: SampleDandiset, bids_examples: str) -> SampleDandiset:
+def bids_dandiset_invalid(
+    new_dandiset: SampleDandiset, bids_examples: str
+) -> SampleDandiset:
```

Comment:
This change was done by `black` (AFAIK) invoked by pre-commit. @jwodder you recently did the reverse change to this elsewhere -- are you using black+pre-commit?

Reply:
I am using pre-commit.  Black will split up long lines, but it won't go in reverse if one of the lines ends in a comma, which was why I had to manually make that previous change.

Reply:
I see! thanks!